ffmpeg_plugin/0001-lavc-svt_hevc-add-libsvt-hevc-encoder-wrapper.patch

```diff
@@ -175,9 +176,10 @@ index 0000000..d9ac04c
 +
 +static void free_buffer(SvtContext *svt_enc)
 +{
-+    uint8_t *in_data = svt_enc->in_buf.pBuffer;
```

Comment:
svt_enc->in_buf.pBuffer points to in_data at line 194. And it is freed here. 
There should not be mem leak.

Reply:
svt_enc->in_buf.pBuffer is set to NULL, when trying to send the last frame to SVT. And we lost the address for memory free.

Reply:
I see.